fdbclient/TenantManagement.actor.cpp

```diff
@@ -37,4 +38,25 @@ TenantMode tenantModeForClusterType(ClusterType clusterType, TenantMode tenantMo
 	}
 }
 
+int64_t extractTenantIdFromMutation(MutationRef m) {
```

Comment:
There is already a function that does this in CommitProxyServer, so we should combine these two into one.